tools/backups2datalad.req.txt

```diff
@@ -1,16 +1,17 @@
 # Python ~= 3.8
-# git-annex >= 8.20210903
+# git-annex >= 10.20220222
```

Comment:
upgrading in `datasets` env: `  git-annex                  8.20211028-alldep_h27987b5_100 --> 10.20220322-alldep_hc98582e_100`